shared/parameters/lob_version.yml

```diff
@@ -7,7 +7,7 @@ lob-version:
 
   description: >
     A string representing the version of the API being used. For more information on versioning,
-    refer to our [Versioning and Changelog]((#tag/Versioning-and-Changelog) documentation.
+    refer to our [Versioning and Changelog](#tag/Versioning-and-Changelog) documentation.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Is this the only change needed to correct the broken link?‏

Reply:
yep - the link was broken - it was trying to take you to `/(#tag` instead of `/#tag`
<img width="388" alt="Screenshot 2024-04-01 at 4 24 24 PM" src="https://github.com/lob/lob-openapi/assets/6289408/0bc15c5e-6732-433b-96e4-a8f5d90b58ee">